### PR TITLE
Merge dev-1.x to 1.x

### PR DESCRIPTION
**Pull request by lvhan028**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Review by lzhangzz — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -22,16 +22,18 @@ workflows:
...
+            docker/.* lint_only true
+            docs/.* lint_only true
             mmdeploy/.* lint_only false
             requirements/.* lint_only false
             tests/.* lint_only false
             third_party/.* lint_only false
             tools/.* lint_only false
-          base-revision: master
+            setup.py lint_only false
+          base-revision: dev-1.x
```
Comment (RunningLeon) [inline .circleci/config.yml:36]: this revision branch could update `1.x`

**Comment by RunningLeon** [comment]
mmroate ut failed. Maybe could be fixed in another PR.

**Review by RunningLeon — APPROVED** [review]
LGTM. Note that this PR better use `Rebase and merge` mode.

File: .circleci/config.yml (hunk 1 above)
Comment (lvhan028) [reply .circleci/config.yml:36]: Will fix it in a new PR

**Pull request merged**